src/server/workbookHandler/WorkbookHandler.js

```diff
@@ -0,0 +1,95 @@
+import fs from 'fs';
+import WorkbookModel from '../database/WorkbookModel.js';
+import FormatError from '../../lib/errors/FormatError.js';
+import ClassConverter from '../../lib/saveWorkbook/ClassConverter.js';
+import JsonConverter from '../../lib/readWorkbook/JsonConverter.js';
+import { zeroID } from '../synchronization/synchronizer.js';
+
+export default class WorkbookHandler {
+  constructor(dataRepo) {
+    this.dataRepo = dataRepo;
+  }
+
+  get(request, response) {
+    if (request.login === undefined || request.login === '') {
+      throw new FormatError('getting workbooks without login');
```

Comment:
Вместо выбрасывания исключения, надо вернуть пользователю соответствующий HTTP статус, в данном случае 403.
```suggestion
res.sendStatus(403);
```

Reply:
Добавь в апи этот статус, я его у get()  не вижу

Reply:
Он там есть. 
```yaml
        "403":
          description: "Недостаточно прав для просмотра книги"
```
Вообще, там ещё есть 401, так что именно для этого случая скорее его следует возвращать.